packages/client/src/components/Modals/GrantDetails.vue

```diff
@@ -92,7 +92,7 @@
         :fields="assignedAgenciesFields"
       >
       <template #cell(actions)="row">
-        <b-button variant="danger" class="mr-1" size="sm" @click="unassignAgenciesToGrant(row)">
+        <b-button v-if="row.item.agency_id === agency.id" variant="danger" class="mr-1" size="sm" @click="unassignAgenciesToGrant(row)">
```

Comment:
I think this should check for membership in `agencies` -- any agency I can assign to I should be able to unassign from, I think (i.e. subagencies of my selected agency).

What if I accidentally assigned to an agency other than my own? As written I don't think this would let me unassign it even if I'm an admin who can change my `selectedAgency` to that agency.

Reply:
Correct. The /session returns user information which has the agency the user belongs. We can use user.agency.subAgencies to determine if the agency is one I have access to. 

But I think the list of agencies we show in the dropdown are already all agencies that i have access to so this wont do anything.

Reply:
Per #61 - only being able to assign to a subagency is a bug. Any user should be able to assign a grant to any other agency.

It sounds like the concern expressed here is "what happens if I assign something on accident"?

We have a WIP feature on the product/design side for a user to be able to track and manage all the grants they've assigned to someone else slated for the next major milestone. In the meantime, the workaround is for users to communicate to the other agency outside of the tool. It's not great, but it's MVP.

@Raphomet wdyt?

Reply:
From further discussion in the standup today:

Since we already track the user who assigned grant, let's go ahead and implement what @mattbroussard proposed - 

**any agency user has assigned, they should also be able to unassign**